examples/cfd-dem/flow_in_bed.prm

```diff
@@ -0,0 +1,141 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set number mesh adapt       = 0
+  set output name              = result_
+  set output frequency	  	= 1
+  set startup time scaling         = 0.6
+  set time end                     = 0.5
+  set time step                    = 0.002
+  set subdivision             = 1 
+  set log precision     = 10 
+  set output path                  	 = ./output/
+end
+
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order            = 1
+    set pressure order            = 1
+end
+
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+    set kinematic viscosity            = 0.00001
+    set density                        = 1
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 				= dealii
+    set grid type            				= subdivided_cylinder
+    set grid arguments       				= 16:0.01:0.1
+    set initial refinement   				= 1
+end
+
+#---------------------------------------------------
+# Void Fraction
+#---------------------------------------------------
+subsection void fraction
```

Comment:
I think this section is a bit poorly named right now. One section is more like CFD-DEM parameters, and the void fraction is another thing right? Maybe just rename the entire section cfd-dem coupling?